pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -235,6 +235,19 @@ func ValidateFlowSchemaResourcePolicyRule(rule *flowcontrol.ResourcePolicyRule,
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("resources"), rule.Resources, "if '*' is present, must not specify other resources"))
 	}
 
+	if len(rule.Namespaces) == 0 {
+		allErrs = append(allErrs, field.Required(fldPath.Child("namespaces"), "resource rules must supply at least one namespace"))
+	} else {
+		for idx, tgtNS := range rule.Namespaces {
+			if tgtNS == flowcontrol.NamespaceEvery || tgtNS == flowcontrol.NamespaceNot {
```

Comment:
nit: I think I would name the constant "NamespaceClusterScope" or something, I know it's oxymoronic but I think "NamespaceNot" might be even less helpful. (e.g. not a namespace along which dimension?)